File: Java/Homework25/view/commands/Command.java
package Homework25.view.commands;

import Homework25.view.Console;

public abstract class Command implements Option{
    private Console console;

    public Command(Console console) {
        this.console = console;
    }
    
    public Console getConsole() {
        return console;
    }

    // public abstract String description();
    // public abstract void execute();
}

File: Python/NotebookProject/view/commands/command.py
from abc import ABC, abstractmethod


class Command(ABC):
    """
        Абстрактный базовый класс для всех команд
        Атрибут:
        ABC (класс): класс ABC из модуля abc, необходимый для создания абстрактных классов."""

    @abstractmethod
    def description(self):
        """Aбстрактный метод, который должен возвращать описание команды."""

    @abstractmethod
    def execute(self):
        """Aбстрактный метод, который должен выполнять действия, связанные с выполнением команды."""